201-customscript-extension-on-ubuntu/README.md

```diff
@@ -0,0 +1,33 @@
+# Simple deployment of an Ubuntu VM with Custom Script extension
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-customscript-extension-on-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template shows a simple example abouht how to deploy [CustomScript extension](https://github.com/Azure/azure-linux-extensions/tree/master/CustomScript) to an Linux VM.
+
+## Parameters
```

Comment:
This table of parameters can be removed now. We will render the description on Azure.com using this readme and this ends up being duplicated.